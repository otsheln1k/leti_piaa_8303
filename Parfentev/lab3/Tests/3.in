10
a
f
a b 7
a c 5
c d 7
b d 2
d e 6
b e 3
d f 8
e f 8
b c 4
a d 4
