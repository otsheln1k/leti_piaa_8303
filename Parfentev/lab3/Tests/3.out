16
a b 7
a c 5
a d 4
b c 2
b d 2
b e 3
c d 7
d e 5
d f 8
e f 8
